app/views/proposals/_proposal_form.html.haml

```diff
@@ -6,7 +6,7 @@
 
     = speaker_selector_input f
 
-    = track_selector_input f
+    = f.input :track, as: :select, collection: @conference.program.tracks, input_html: { class: 'select-help-toggle' }, include_blank: '(Please categorize your event)' if @conference.program.tracks.any?
```

Comment:
Can you please explain this change here?
Why are changing the code for the collection of the available tracks?

Also one line below, the equivalent text for difficulty level is '(Please select)'. We should not say the same thing to our users in different ways.

Reply:
@differentreality I got it and fixed it! Please have a look!